docs/sql-ref-syntax-ddl-create-function.md

```diff
@@ -19,4 +19,148 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`CREATE FUNCTION` statement is used to create a temporary or permanent function
+in Spark. Temporary functions are scoped at a session level where as permanent
+functions are created in the persistent catalog and are made available to
+all sessions. The resources specified in the `USING` clause are made available
+to all executors when they are executed for the first time.
```

Comment:
We also need to explain we can create/register temporary SQL functions via Python/Scala/Java APIs.

Reply:
@gatorsmile I have created a place holder link for custom scalar functions. There is already a place holder for aggregate functions and a lot of content is already in place.